src/python/bcc/__init__.py

```diff
@@ -62,13 +62,47 @@ def resolve_name(self, name):
             return -1
         return addr.value
 
+class BPFPerfType:
```

Comment:
I think this should probably be just `PerfType`, which aligns better with other classes not having the `BPF` prefix.